src/pages/Profiles.vue

```diff
@@ -399,36 +379,8 @@ export default class Profiles extends Vue {
         this.renamingProfile = false;
     }
 
-    removeProfile() {
-        this.removingProfile = true;
-    }
-
-    async removeProfileAfterConfirmation() {
-        try {
-            await FileUtils.emptyDirectory(this.activeProfile.getPathOfProfile());
-            await fs.rmdir(this.activeProfile.getPathOfProfile());
-        } catch (e) {
-            const err = R2Error.fromThrownValue(e, 'Error whilst deleting profile');
-            this.$store.commit('error/handleError', err);
-        }
-        if (
-            this.activeProfile
-                .getProfileName()
-                .toLowerCase() !== 'default'
-        ) {
-            for (let profileIteration = 0; profileIteration < this.profileList.length; profileIteration++) {
-                if (this.profileList[profileIteration] === this.activeProfile.getProfileName()) {
-                    this.profileList.splice(profileIteration, 1);
-                    break;
-                }
-            }
-        }
-        await this.setSelectedProfile('Default');
```

Comment:
The new implementation doesn't call this line anymore. (Well, Vuex can't call component methods, and while child components can emit events to parents, we don't want to do that in this case.) This creates a bug that can reproduced like this:

1. Install some mods in default profile
2. Create a second profile and install different mods
3. Return to profile selection
4. Delete the second profile, "selected profile" automatically changes to default
5. Clicking "Select profile" takes you to profile mod listing, which shows the content of the second, now deleted profile instead of the default profile

I *think* best way to fix this would be something like:

1. Copy the implementation of `Profiles.setSelectedProfile` to `ProfilesModule` as a separate action method (but as discussed earlier, leave out the try-catch and the error modal stuff)
  * Do this as a separate action since other places will also be calling it later on
2. Instead of `removeSelectedProfile` action calling `profile/updateActiveProfile`, call the new action instead (with prewarm cache)
  * We could alternatively call the new action from `DeleteProfileModal.removeProfile`, but I'm like 60% sure the suggestion above is a bit nicer